cuebot/src/main/java/com/imageworks/spcue/PrometheusMetricsCollector.java

```diff
@@ -119,6 +123,52 @@ public class PrometheusMetricsCollector {
             .labelNames("env", "cuebot_host", "render_node", "job_name", "frame_name", "frame_id")
             .register();
 
+    // Monitoring event metrics
+    private static final Counter monitoringEventsPublishedCounter =
+            Counter.build().name("cue_monitoring_events_published_total")
+                    .help("Total number of monitoring events published to Kafka")
+                    .labelNames("env", "cuebot_host", "event_type").register();
+
+    private static final Counter monitoringEventsDroppedCounter =
+            Counter.build().name("cue_monitoring_events_dropped_total")
+                    .help("Total number of monitoring events dropped due to queue overflow")
+                    .labelNames("env", "cuebot_host", "event_type").register();
+
+    private static final Gauge monitoringEventQueueSize =
+            Gauge.build().name("cue_monitoring_event_queue_size")
+                    .help("Current size of the monitoring event publishing queue")
+                    .labelNames("env", "cuebot_host").register();
+
+    private static final Gauge elasticsearchIndexQueueSize =
+            Gauge.build().name("cue_elasticsearch_index_queue_size")
+                    .help("Current size of the Elasticsearch indexing queue")
+                    .labelNames("env", "cuebot_host").register();
```

Comment:
Verify if this metric is really useful. It is basically monitoring the monitoring system.